actions/users

```diff
@@ -72,17 +67,9 @@ def configure_slapd():
 
 def configure_ldap_structure():
     """Configure LDAP basic structure."""
-    was_running = action_utils.service_is_running('slapd')
-    if not was_running:
-        action_utils.service_start('slapd')
-
-    try:
-        setup_admin()
-        create_organizational_unit('users')
-        create_organizational_unit('groups')
-    finally:
-        if not was_running:
-            action_utils.service_stop('slapd')
+    setup_admin()
+    create_organizational_unit('users')
+    create_organizational_unit('groups')
```

Comment:
Should this safety check be removed?

Reply:
My assumption was that this setup would be triggered by plinth when it normally starts (probably on first boot). It would be unusual for slapd to not be running at this time.

That said, we can keep the safety check, to check if slapd is running, and start it if needed. I guess this will keep compatibility with the old setup approach. But I would still remove the "finally" block--we want to keep slapd running after setup completes.

Reply:
- During a normal system startup, if Plinth is started before slapd, and if we start the service instead of systemd, I wonder what would happen.  Would that effect the way systemd manages the service?
- We should add the following to our systemd service file:
   ```
   Wants=slapd.service
   After=slapd.service
   ```
- After this, it should be okay to remove the safety entirely.  Until then, I shall keep the start part and remove the try/finally.

I currently building an image with the changes to test fully and I will commit soon.